src/components/Tools/ProgressTracker.vue

```diff
@@ -124,6 +124,22 @@ export default defineComponent({
       }
     },
   },
+  methods: {
+    generateRequirementProgressString(req: string, reqGroup: string): string {
+      switch (reqGroup) {
+        case 'College':
+          return getCollegeAbbrev(req) + ' ' + reqGroup + ' Requirements';
+        case 'Major':
+          return getMajorAbbrev(req) + ' ' + reqGroup + ' Requirements';
+        case 'Minor':
+          return getMinorAbbrev(req) + ' ' + reqGroup + ' Requirements';
+        case 'Grad':
+          return getGradAbbrev(req) + ' ' + reqGroup + ' Requirements';
+        default:
+          return '';
```

Comment:
Can we refactor this big switch to use a function that returns the correct abbreviation given a requirement?